app/views/contexts/summary/answer.py

```diff
@@ -86,16 +88,24 @@ def _return_to_answer_id(
         return_to: str | None,
         list_item_id: str | None,
         is_in_repeating_section: bool,
+        return_to_answer_id: str | None,
     ) -> str | None:
         """
         If the summary page using this answer has repeating answers, but it is not in a repeating section,
         then the answer ids will be suffixed with list item id, so the return to answer id link also needs this to work correctly
         """
+        answer_id = None
```

Comment:
to fix the below this will need to be `: str | None`

Reply:
I fixed the below by nesting the logic, so this one is not needed?

Reply:
I thought mypy might infer type as None and then not like you adding a string to it later, but if its not complaining then all good 👍